.gitignore

```
@@ -20,3 +20,6 @@ DerivedData
 .swiftpm/
 generated/
 *coverage.txt
+
+# We can generate the project from the package. No need to check it in.
+CacheAdvance.xcodeproj/
```

Smart thing to ignore. We don’t need any xcode projects: should we ignore any project, no matter where it is located?
```suggestion
*.xcodeproj/
```

---

I was on the fence about this. Let's do what you suggested.
I'll simplify the comment a bit too.